src/components/ScaleLattice.vue

```diff
@@ -0,0 +1,340 @@
+otonalFundamental
+<script setup lang="ts">
+import type { Interval, Scale } from "scale-workshop-core";
+import {
+  computed,
+  onMounted,
+  type ComputedRef,
+  ref,
+  onBeforeUnmount,
+} from "vue";
+import { PRIMES } from "xen-dev-utils";
+import { monzoEuclideanDistance, getPairCombinations } from "@/utils";
+const props = defineProps<{
+  scale: Scale;
+  width: number;
+  height: number;
+  heldScaleDegrees: Set<number>;
+}>();
+
+type Coord = { x: number; y: number };
+type Node = {
+  x: number;
+  y: number;
+  label: string;
+  index: number;
+  fill: string;
+};
+
+type Coords = {
+  [key: string]: Coord;
+};
+
+const coords: Coords = {
```

Comment:
I'd prefer this to be an array corresponding to monzo indices with comments indicating which prime we're dealing with. There's no need to map monzo indices to primes to strings to object keys.